packages/maskbook/src/plugins/Trader/apis/1inch/index.ts

```diff
@@ -0,0 +1,36 @@
+import BigNumber from 'bignumber.js'
+import { first } from 'lodash-es'
+import { BIPS_BASE, ONE_INCH_BASE_URL } from '../../constants'
+import type {
+    SwapQuoteOneResponse,
+    SwapQuoteOneRequest,
+    SwapOneServerErrorResponse,
+    SwapOneValidationErrorResponse,
+    SwapOneErrorResponse,
+} from '../../types'
+import type { NetworkType } from '@masknet/web3-shared'
+import urlcat from 'urlcat'
+
+export async function swapOneQuote(request: SwapQuoteOneRequest, networkType: NetworkType) {
+    const params: Record<string, string> = {}
+    Object.entries(request).map(([key, value]) => {
+        if (typeof value === 'string') params[key] = value
+    })
+    if (request.slippage) params.slippage = new BigNumber(request.slippage).dividedBy(BIPS_BASE).toFixed()
+
+    const response =
+        (await fetch(urlcat(ONE_INCH_BASE_URL[networkType], 'swap', params))) ||
+        (await fetch(urlcat(ONE_INCH_BASE_URL[networkType], 'quote', params)))
```

Comment:
Why are you're doing this? This doesn't make sense.

Reply:
I fixed, now i am using swap endpoint